### PR TITLE
First implementation

### PR DESCRIPTION
**Pull request by sunbo57123**
This PR comes from [ros2cli#532](https://github.com/ros2/ros2cli/pull/532), the background is creating meta-package for ros2cli.

### CODE REVIEW

File: ros2cli_common_extensions/package.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ros2cli_common_extensions</name>
+  <version>0.1.0</version>
```
Comment (dirk-thomas) [inline ros2cli_common_extensions/package.xml:5]: This should start with 0.0.0 in order to be bumped for the first release.

File: ros2cli_common_extensions/package.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  <exec_depend>ros2node</exec_depend>
+  <exec_depend>ros2param</exec_depend>
+  <exec_depend>ros2pkg</exec_depend>
+  <exec_depend>ros2run</exec_depend>
+  <exec_depend>ros2service</exec_depend>
+  <exec_depend>ros2topic</exec_depend>
+  <exec_depend>sros2</exec_depend>
+  <exec_depend>ros2launch</exec_depend>
+  <exec_depend>launch_xml</exec_depend>
+  <exec_depend>launch_yaml</exec_depend>
```
Comment (dirk-thomas) [inline ros2cli_common_extensions/package.xml:28]: Nitpick: commonly we order the dependency within one group alphabetically.

**Comment by dirk-thomas** [comment]
@ivanpauno Since you are listed as the maintainer can you run CI for this new package, add it to the `ros2.repos` file after this has been merged and release the package (probably into Rolling as well as Foxy?).

**Comment by Blast545** [comment]
CI results:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/11425/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/6641/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/9368/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/11328/)

**Comment by ivanpauno** [comment]
> @ivanpauno Since you are listed as the maintainer can you run CI for this new package, add it to the `ros2.repos` file after this has been merged and release the package (probably into Rolling as well as Foxy?).

I've made a release into Rolling (https://github.com/ros/rosdistro/pull/25788).
I can also make one for Foxy, but I'm not sure if that's needed. @jacobperron WDYT?

I can also create a PR to simplify the [variants](https://github.com/ros2/variants/blob/master/ros_core/package.xml).

**Comment by jacobperron** [comment]
> I can also make one for Foxy, but I'm not sure if that's needed. @jacobperron WDYT?

It's not needed, but I think it is nice to have and relatively low risk.

**Pull request merged**